test/configuration_server_test.erl

```diff
@@ -0,0 +1,40 @@
+-module(configuration_server_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-include("configuration_server.hrl").
+
+default_directories_test() ->
+    ?assertEqual(?CONFIG_PATHS, configuration_server:default_directories()).
+
+default_filenames_test() ->
+    ?assertEqual(?FILE_NAMES, configuration_server:default_filenames()).
```

Comment:
I'm the biggest proponent of TDD, but this just strikes me as awkward. This is basically testing implementation details, which is an anti-pattern imho.

Reply:
Ok, i'll drop it.